### ISSUE TITLE
little tiles redirect conflict

### ISSUE THREAD

**Issue by fayer3** [ferriarnus/VivecraftForge#278]
little tiles crashes because of a redirect conflict, probably fixable with mixin extras
```
@Redirect conflict. Skipping littletiles.mixins.json:client.MinecraftMixin->@Redirect::isDestroying(Lnet/minecraft/client/multiplayer/MultiPlayerGameMode;)Z with priority 1000, already redirected by vivecraft.mixins.json:client_vr.MinecraftVRMixin->@Redirect::vivecraft$seatedCheck(Lnet/minecraft/client/multiplayer/MultiPlayerGameMode;)Z with priority 1000
```

https://github.com/CreativeMD/LittleTiles/blob/3b191a2e52bf96e979cb0cb3f1766841b77630df/src/main/java/team/creative/littletiles/mixin/client/MinecraftMixin.java#L89-L92

https://github.com/Vivecraft/VivecraftMod/blob/c1814bf0891bfdd5575ace7d00acd966af220f3a/common/src/main/java/org/vivecraft/mixin/client_vr/MinecraftVRMixin.java#L443

https://modrinth.com/mod/littletiles

**Comment by CreativeMD** [comment]
Made use of MixinExtra's wrapper operator, which should hopefully fix this issue.

**Comment by fayer3** [comment]
#284 will replace all redirects with wrap operations which will also fix this

**Comment by fayer3** [comment]
should be fixed with #284
